Interfaces/API/Dirac.py

```diff
@@ -1262,6 +1282,119 @@ def getLfnMetadata(self, lfns, printOutput=False):
 
     return repsResult
 
+  #### User Metadata API ######################################################
```

Comment:
What are the advantages of adding these methods?

Reply:
I'm not insisting here. I just wanted to add all the client-side API call to already exported server-side calls, assuming that someone you had exported them had a reason to do so.